app/uk/gov/hmrc/accessibilitystatementfrontend/views/AccessibilityStatementTemplate.scala.html

```diff
@@ -60,11 +60,14 @@ <h2 class="govuk-heading-l">@{messages("accessibility.heading")}</h2>
 } else {
     <p class="govuk-body">{messages("accessibility.has-issues")}</p>
 } }
-<ul class="govuk-list govuk-list--bullet">
-@for(problem <- statement.accessibilityProblems) {
-    <li>@problem</li>
+
+@if(statement.accessibilityProblems.nonEmpty) {
```

Comment:
Being a test pedant, but would it be worth adding one? :-)

Reply:
There is already a test that accessibility items render as `<li>`, are you thinking a test that `<ul>` isn't rendered for `accessibilityProblems.isEmpty?`